api/internal/owner/serializers.py

```diff
@@ -236,6 +236,7 @@ def get_scheduled_phase(self, schedule):
                 extra=dict(
                     ownerid=schedule.metadata.obo_organization,
                     requesting_user_id=schedule.metadata.obo,
+                    phases=schedule["phases"],
```

Comment:
Are we sure this key exists? otherwise it will throw an error 

You can try, `phases=schedule.get("phases", "no phases"))` or something similar

Reply:
I believe this will only be called if phases exist, but I'll add your changes anyway - can't hurt 🙂